projects/plugins/jetpack/class-jetpack-recommendations-banner.php

```diff
@@ -245,14 +245,16 @@ public function render_banner() {
 						</mask><g mask="url(#jp-dismiss-mask0)"><rect x="0.582031" width="23.8823" height="24" /></g></svg>
 					<span><?php esc_attr_e( 'Dismiss', 'jetpack' ); ?></span>
 				</div>
-				<img
-						src="<?php echo esc_url( plugins_url( 'images/recommendations/background.svg', JETPACK__PLUGIN_FILE ), 'jetpack' ); ?>"
-						class="jp-recommendations-banner__illustration-background"
-				/>
-				<img
-						src="<?php echo esc_url( plugins_url( 'images/recommendations/site-type-illustration.jpg', JETPACK__PLUGIN_FILE ), 'jetpack' ); ?>"
+				<picture>
+					<source
+						type="image/webp"
+						srcset="<?php echo esc_url( $this->img_path( 1, 'webp' ) ); ?> 1x, <?php echo esc_url( $this->img_path( 2, 'webp' ) ); ?> 2x">
+					<img
 						class="jp-recommendations-banner__illustration-foreground"
-				/>
+						srcset="<?php echo esc_url( $this->img_path( 2 ) ); ?> 2x"
+						src="<?php echo esc_url( $this->img_path() ); ?>"
+						alt="">
+				</picture>
```

Comment:
Part of me wonders whether we should use Photon instead of shipping different versions of the same image? This way we would let Photon handle webp and srcset for us.

Reply:
Is this blocking, or an improvement @jeherve?

Reply:
Definitely not a blocker imo, but a possible improvement. I think we could indeed stick with one version of the image, and serve it with Photon all the time, even when the module is disabled. The module really is about the images on the frontend of the site.